src/csharp/Grpc.Tools/ProtoCompile.cs

```diff
@@ -206,6 +206,63 @@ public class ProtoCompile : ToolTask
                 }
             },
 
+            // Example warning from plugins that use GOOGLE_LOG
+            // [libprotobuf WARNING T:\altsrc\..\csharp_enum.cc:74] Duplicate enum value Work (originally Work) in PhoneType; adding underscore to distinguish
+            new ErrorListFilter
+            {
+                Pattern = new Regex(
+                    pattern: "^\\[.+? WARNING (?'FILENAME'.+?):(?'LINE'\\d+?)\\] ?(?'TEXT'.*)",
+                    options: RegexOptions.Compiled | RegexOptions.IgnoreCase,
+                    matchTimeout: s_regexTimeout),
+                LogAction = (log, match) =>
+                {
+                    // The filename and line logged by the plugins may not be useful to the
+                    // end user as they are not the location in the proto file but rather
+                    // in the source code for the plugin. Log them anyway as they may help in
+                    // diagnositics.
```

Comment:
Nit: diagnositics => diagnostics, here and below

Reply:
fixed typos